### DISCUSSION TITLE
FLAC files are not played

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1392] [General]
Hi,

Yesterday I had an unusual thing. I imported some flac files, but they were not recognized as such by the r/audio database, only the (empty) matching directory.

I checked the files with xACT (macOS) successfully and they are also playable on all other devices (PC/mobile devices). Just not with r/Audio. 

If I decode the files as WAV files, it works. Also a re-encoding does not help.

What can be the reason for this?

PS: my heart is not in it, I can also live with WAV files ;-)

Oliver

**Comment by R101G** [comment]
@master-kw 

Is ffmpeg enabled?

**Reply by cmh714** [reply]
you dont need ffmpeg for FLAC.

**Reply by kw-sudo** [reply]
To precise, it was only one complete album. The other 900 are working perfectly. 

That’s the reason for asking. In the past re-encoding has helped in such a case.

**Reply by cmh714** [reply]
Yeah not sure where the problem is.....with Raudio or with the flac encoding. My 6TB is ALL FLAC and I do not use ffmpeg so thats not the issue. Maybe try re-encoding, or try with another album, or with something that you know worked....GL!!

**Reply by kw-sudo** [reply]
As I said before, this is an isolated case. All other files were imported correctly. 

I would have been interested in a possible cause, especially since a check (checksum/flac-fix) detected no errors and a re-encoding was unsuccessful.

Oliver

**Reply by EarsofTin** [reply]
Do you have a .cue or .m3u file in the directory? The presence of them prevent me from playing flac or mp3 files with rAudio although LMS works with them.

**Reply by kw-sudo** [reply]
„Same procedure as every time.“ No cue , m3u or embedded lyrics, covers. Just ‚clean‘ flac files and seperate cover. 

Oliver

**Comment by kw-sudo** [comment]
Forget it, guys. The problem was somewhere else: it was the cover file itself in the directory that caused the problem (not embedded!). 

After I imported the files without covers and updated the database, r/Audio also properly embedded the album. A new cover file solved it.

Thanks all the way!
Oliver
